techbuzz/src/main/java/com/sivalabs/techbuzz/users/adapter/repositories/JooqUserRepository.java

```diff
@@ -33,6 +34,14 @@ public Optional<UserProfile> findProfileById(Long id) {
                 .fetchOptional(Records.mapping(UserProfile::new));
     }
 
+    public Optional<String> findVerifiedUsersMailIds() {
```

Comment:
Getting the emails as comma separated string is not persistence related logic. From this method we should return `List<String>` and before sending email we should convert it into comma separated string.